### PR TITLE
feat: Upgrade from brunch to webpack

### PR DESCRIPTION
**Pull request by losvedir**
#### Summary of changes
**Asana Ticket:** [💳 Migrate Prediction Analyzer to use webpack](https://app.asana.com/0/584764604969369/1130700386587542)

I tried to do this iteratively, but got flummoxed finding reasonable commit points and ended up just doing it en masse. I stole most of the config from Arrow.

This means it uses TypeScript as the compiler, though I annotated as minimally as possible. It also included the various linting/formatting checks, so the code was updated to agree with those (as well as the `static_analysis.sh` script to ensure PRs abide by them).

#### Reviewer Checklist
- [ ] Meets ticket's acceptance criteria
- [ ] Any new or changed functions have typespecs
- [ ] Tests were added for any new functionality (don't just rely on Codecov)

### CODE REVIEW

File: assets/brunch-config.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ exports.config = {
   // See http://brunch.io/#documentation for docs.
   files: {
     javascripts: {
-      joinTo: "js/app.js"
+      joinTo: "js/app.js",
```
Comment (lemald) [inline assets/brunch-config.js:5]: Do we still need `brunch-config.js` at all?

Comment (losvedir) [reply assets/brunch-config.js]: Nope! Good catch.

**Comment by losvedir** [comment]
This branch (`10117d7`) has now been deployed.

**Comment by losvedir** [comment]
Okay, *now* it's been deployed.

**Pull request merged**